seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -696,15 +696,16 @@
       "id": "bad87fee1348bd9aedf08807",
       "title": "Import a Google Font",
       "description": [
-        "Now, let's import and apply a Google font (note that if Google is blocked in your country, you will need to skip this challenge).",
+        "In addition to specifying common fonts that are found on most operating systems, we can also specify non-standard, custom web fonts for use on our website. There are varies type foundries on the web but, for this example we will focus on the Google Fonts library.",
```

Comment:
~There are varies type foundries on the web but,~

There are various sources for web fonts on the internet but,